.ci/Jenkinsfile

```diff
@@ -392,6 +430,21 @@ def generateStep(Map params = [:]){
   }
 }
 
+def prepareRelease(String nodeVersion='node:lts', Closure body){
+  withNpmrc(secret: "${env.NPMRC_SECRET}", path: "${env.WORKSPACE}/${env.BASE_DIR}") {
+    withCredentials([usernamePassword(credentialsId: '2a9602aa-ab9f-4e52-baf3-b71ca88469c7-UserAndToken',
```

Comment:
@v1v I need some direction in getting the right token here.

Reply:
I think that's the correct one. There is only one token for the `apm` user, I'd rather keep the name obfuscated to avoid any disclosure.

Reply:
OK, cool. Wasn't sure what was what but this sounds like it should work. Thanks!